plugins/catalog/src/alpha.ts

```diff
@@ -15,3 +15,40 @@
  */
 
 export * from './translation';
+
+import {
+  DiscoveryApi,
+  FetchApi,
+  discoveryApiRef,
+  fetchApiRef,
+} from '@backstage/core-plugin-api';
+import { CatalogClient } from '@backstage/catalog-client';
+import {
+  createApiExtension,
+  createPlugin,
+} from '@backstage/frontend-plugin-api';
+import { catalogApiRef } from '@backstage/plugin-catalog-react';
+
+/** @alpha */
+export const CatalogApi = createApiExtension({
```

Comment:
>[!Note]
The search plugin uses Catalog API, so created it to provide a default implementation.